tests/nexus_workflow_test.go

```diff
@@ -2249,7 +2249,7 @@ func (s *NexusWorkflowTestSuite) TestNexusCallbackAfterCallerComplete() {
 	svc := nexus.NewService("test")
 
 	handlerWF := func(ctx workflow.Context, _ nexus.NoValue) (nexus.NoValue, error) {
-		return nil, workflow.Sleep(ctx, 1*time.Second)
+		return nil, workflow.Sleep(ctx, 2*time.Second)
```

Comment:
Why is this required? The workflow isn't part of the `StartNexusOperation` request duration, it is run asynchronously and completed via a callback.

Reply:
This timer isn't part of the request flow and is not subject to the changes in this PR.